.env.example

```diff
@@ -26,3 +26,5 @@ PUBLIC_OTP_SERVER_URL=""
 PUBLIC_ANALYTICS_DOMAIN=""
 PUBLIC_ANALYTICS_ENABLED=true
 PUBLIC_ANALYTICS_API_HOS=""
+
```

Comment:
In the future, please be sure to avoid including extra whitespace like this in a file that you've reverted your other changes in.